.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you .gitignore that ?

Reply:
It's already ignored in the PR #87. I delete it from this one.